mycroft/audio/service.py

```diff
@@ -75,7 +78,91 @@ def __init__(self, ready_hook=on_ready, error_hook=on_error,
             LOG.exception(e)
             self.status.set_error(e)
 
+    @staticmethod
+    def get_tts_lang_options(lang, blacklist=None):
+        blacklist = blacklist or []
+        opts = []
+        cfgs = get_tts_lang_configs(lang=lang, include_dialects=True)
+        for engine, configs in cfgs.items():
+            if engine in blacklist:
+                continue
+            # For Display purposes, we want to show the engine name without the underscore or dash and capitalized all
+            plugin_display_name = engine.replace("_", " ").replace("-", " ").title()
+            for voice in configs:
+                voice["plugin_name"] = plugin_display_name
+                voice["engine"] = engine
+                voice["lang"] = voice.get("lang") or lang
+                opts.append(voice)
+        return opts
+
+    @staticmethod
+    def get_g2p_lang_options(lang, blacklist=None):
+        blacklist = blacklist or []
+        opts = []
+        cfgs = get_g2p_lang_configs(lang=lang, include_dialects=True)
+        for engine, configs in cfgs.items():
+            if engine in blacklist:
+                continue
+            # For Display purposes, we want to show the engine name without the underscore or dash and capitalized all
+            plugin_display_name = engine.replace("_", " ").replace("-", " ").title()
+            for voice in configs:
+                voice["plugin_name"] = plugin_display_name
+                voice["engine"] = engine
+                voice["lang"] = voice.get("lang") or lang
+                opts.append(voice)
+        return opts
+
+    @staticmethod
+    def get_audio_options(blacklist=None):
+        blacklist = blacklist or []
+        opts = []
+        cfgs = get_audio_service_configs()
+        for engine, configs in cfgs.items():
+            if engine in blacklist:
+                continue
+            # For Display purposes, we want to show the engine name without the underscore or dash and capitalized all
+            plugin_display_name = engine.replace("_", " ").replace("-", " ").title()
+            for voice in configs:
+                voice["plugin_name"] = plugin_display_name
+                voice["engine"] = engine
+                opts.append(voice)
+        return opts
+
+    def handle_opm_tts_query(self, message):
```

Comment:
Consider adding a docstring that includes the message type associated with this handler